### DISCUSSION TITLE
2.1.0

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#10998] [General]
*  Enable DNS tunneling by default
*  Skip installing the MSIX for the current user when running as SYSTEM (solves #10906)
*  Only run distro init logic when started from mini_init (solves #10883)
*  Create an execution context for MSI package installation via wslinstaller
*  Configure all executables as LSP_SYSTEM to avoid loading proxy dll's (solves #4177)
*  Add logic to remove leaked MSIX service registration (solves #10831)
*  Log WorkingDirectory when CreateProcess fails (solves #6170)
*  Close the distro socket fd before starting the system distro (solves #10866)
*  Use the MSI API to install & upgrade instead of msiexec
*  Fix some issues in printf format specifiers for logging
*  Update Microsoft.Windows.ImplementationLibrary version 1.0.240102.2
*  Update Wix to version 4.0.3
*  Implement DrvFs Perf notification

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/2.1.0'>2.1.0</a>.</em>

**Comment by tinlittle** [comment]
Missing from the release notes:
Kernel upgraded to 5.15.137 from 5.15.133.

p.s. curiously to know when will wsl2 upgrade the kernel to 6.1 by default or maybe upgrade to 6.6 is planned in the near future and 6.1 will be skipped?

**Reply by benhillis** [reply]
Good question - we're still trying to sort out a what seems like a regression in the storvsc driver which is causing seemingly random corruptions when reading from the VHD. Once we figure that out, we will update the kernel to 6.1

**Comment by muratyurdakul75** [comment]
Hi

There are 2 NUMA on servers or some computers. Is it planned to bring support for 2 NUMA? It's currently using only 1 of 2 CPUs.
